cloudbuild.yaml

```diff
@@ -30,9 +30,14 @@ steps:
   id: lint
 - name: "make-docker"
   dir: "build"
-  args: [ "test" ] # run tests
+  args: [ "-j", "3", "test" ] # run tests, in parallel
   waitFor: ['lint']
   id: tests
+- name: "make-docker" # push a preview of the website
+  dir: "build"
+  args: ["site-deploy-preview"]
```

Comment:
I have some question about this how does that work ? do you host a static site for every PR or only one for all ? what it the URL ? Should you update the github build message to get a link for that preview ?